### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by cclarkson**
**UPDATE**
This PR now contains a full draft of the paper - ready for review.

### CODE REVIEW

**Comment by cclarkson** [comment]
@EricRLucas  and @hardingnj - if someone could re-write the intro first line I'd be grateful. It's been through so many revisions it feels super clunky now (even though it finally conveys what I wanted it to).

File: main.tex [hunk 1]
```diff
@@ -70,19 +70,102 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section*{Introduction}
 
+
 %%
-TODO
+During the 1950s, the widespread use of the insecticide DDT and the antimalarial drug chloroquine in the fight against malaria resulted in high levels of insecticide and drug resistance, demonstrating both the strength that anthropogenic pressure can exert and the speed at which selection can generate phenotypes to counteract it.
+%
+This pattern has been repeated for over half a century as new insecticides and antimalarials have been developed, deployed, then rendered ineffective by resistance in an evolutionary 'arms race'.
```
Comment (EricRLucas) [inline main.tex]: Is "ineffective" a defensible statement for insecticides? If so, I think we need some citations here to support that.

File: main.tex [hunk 2]
```diff
@@ -70,19 +70,102 @@
...
+%
+However, the last 15 years witnessed a change in fortunes, mass bed-net distributions and artemisinin combination therapy (ACT) saw large reductions in malaria morbidity and mortality \cite{Bhatt2015}.
```
Comment (EricRLucas) [inline main.tex]: I feel it wasn't so much a "change in fortunes" as an increase in effort.

File: main.tex [hunk 3]
```diff
@@ -70,19 +70,102 @@
...
+%
+In 2018 alone, 172 million long-lasting insecticide treated nets (LLINs) distributed for free in sub-Saharan Africa (www./allianceformalariaprevention.com/net-mapping-project/).
```
Comment (EricRLucas) [inline main.tex]: *were* distributed

File: main.tex [hunk 4]
```diff
@@ -70,19 +70,102 @@
...
+%%
+Currently, surveillance of insecticide resistance in natural malaria vector populations involves two approaches:
+%
+resistance susceptibility testing with insecticide coated paper (\textit{e.g.} WHO tube assay \cite{world2016test}) and the detection of genetic loci associated with resistance via molecular assays (\textit{e.g.} PCR \cite{Ranson2000} and TaqMan \cite{bass2007}).
+%
+Measuring the insecticide susceptibility phenotypes of live mosquitoes samples provides an almost real-time picture of the strength of resistance (perhaps just one generation removed) and insecticides can be tested with no prior knowledge about potential modes of resistance or of underlying causative genetic resistance loci.
+%
+Monitoring natural populations is important but this approach is laborious, plus it requires expensive environmentally controllable spaces (humidity/temperature/light) to enable mosquito breeding and to ensure repeatability of testing \cite{kleinschmidt2018}.
+%
+Detecting a resistance phenotype within a mosquito population using this methodology may also be difficult unless resistance is present at high frequencies (Lucas 2019 - Bioxiv).
```
Comment (EricRLucas) [inline main.tex]: Not sure about using this citation here, as it's not really a conclusion of the paper, nor do we really build a strong, well-supported argument for it. It was a sentence that Martin suggested that we add, but he didn't provide a reference for it and I haven't been able to find one. Maybe leave it unreferenced? Or take out the sentence? Or find a better reference (though I tried and failed at that).

File: main.tex [hunk 5]
```diff
@@ -70,19 +70,102 @@
...
+%%
+
+
+%%
+Molecular insecticide resistance assays can, conversely, be run quickly and cheaply on dead samples collected directly from natural populations without requirement for breeding the high numbers of mosquitoes necessary for resistance assays.
+%
+The specificity of molecular assays is useful, as the frequency of a specific genetic locus contributing to resistance can be measured within a population, although this is also restrictive, as only known resistance markers can be tested using this method.
+%
+Historically the development and validation of new molecular markers is slow, though several have been identified for \textit{Anopheles gambiae/coluzzii} in the last 20 years \cite{du2005, jones2012, martinez1998, mitchell2014, ranson2000, riveron2014, weetman2018, weill2003}.
```
Comment (EricRLucas) [inline main.tex]: is slow -> has been slow

File: main.tex [hunk 6]
```diff
@@ -70,19 +70,102 @@
...
+%
+However, these markers explain just a fraction of the variance in resistance phenotype \cite{donnelly2009, donnelly2016}, for example a combination of three molecular markers were able to explain only ~50\% of the variation of resistance to the insecticide DDT \cite{mitchell2014}.
```
Comment (EricRLucas) [inline main.tex]: "just a fraction" makes it sound tiny, but then you give an example where it's 50\% (admittedly still a fraction, but probably not what the reader envisaged)

File: main.tex [hunk 7]
```diff
@@ -70,19 +70,102 @@
...
+If we are to improve our understanding of insecticide resistance evolution in malaria vectors, an alternative approach is required to both find this missing variation and to better monitor resistance levels in country. 
+%%
+
+
+%%
+The \textit{Anopheles gambiae} 1000 Genomes Project (Ag1000G), an international collaboration, was established with the aim of generating a catalogue of deep-sequencing-generated genomic variation from natural mosquito populations to better understand and thus more effectively combat the principal African malaria vectors. 
+%
+The project forms part of the Malaria Epidemiology Network (MalariaGen - www.malariagen.net) a community of data-sharing and researchers investigating how genetic variation in humans, malaria parasites and malaria vectors can inform biology, epidemiology and ultimately global malaria eradication.
+%
+The output of Ag1000G includes whole genome variation, phased haplotypes, meta-data and genomic scans for signatures of selection, all of which is freely available to analyse under MalariaGen's guiding principles on data sharing \cite{network2008, chokshi2006, parker2009}.
```
Comment (EricRLucas) [inline main.tex]: is freely available -> are freely available

File: main.tex [hunk 8]
```diff
@@ -70,19 +70,102 @@
...
+%
+@@Do we have anything else from the modelling guys understandingwe can put here?@@.
+%
+Recent research into the metabolic mode of insecticide resistance, made possible through the Ag1000g's deep sequencing data, has also elucidated previously unknown complexity of copy number variation which may hold the key to the discovering the 'missing' variation apparent when resistance association is measured \cite{lucas2019, mitchell2014}.
+%
+It has become evident that datasets which can be used to quickly and cheaply to identify potential markers of resistance or improve the vector control 'genetic toolkit' are essential to augment current vector control efforts.
+%
+A deeper understanding of resistance evolution and the ability to parameterise models for resistance spread provided by genomic surveillance of natural populations facilitates a move into predictive rather than reactive insecticide resistance management.
```
Comment (EricRLucas) [inline main.tex]: I found this sentence difficult to read. In particular, it wasn't clear what "provided by genomic surveillance of natural populations" was referring to.

File: main.tex [hunk 9]
```diff
@@ -70,19 +70,102 @@
...
+%%
+
+
+%%
+The pervasive increase in pyrethroid resistance detected across Sub-Saharan Africa is raising fears of LLIN failures leading to increases in malaria \cite{churcher2016, Hemingway2016}.
+%
+To ameliorate the risk of vector control failure, pyrethoid LLINs containing the synergist piperonyl butoxide (PBO) have been in development and testing for some time \cite{darriet2011, protopopoff2018}.
+%
+PBO enhances the potency of certain insecticides (including pyrethroids), where resistance to these chemicals is cytochrome P450 mediated (metabolic resistance), though where resistance is purely target-site mediated (\textit{e.g. kdr} mutations) gains are less likely to be seen.
```
Comment (EricRLucas) [inline main.tex]: Personal preference: I would write "mediated by cytochrome P450s"

File: main.tex [hunk 10]
```diff
@@ -70,19 +70,102 @@
...
+%
+A study using  meta-analysis to parameterise predictive mathematical models suggests that PBO bednets could avert up to 0.5 clinical cases per person per year in some locations \cite{churcher2016}.
```
Comment (EricRLucas) [inline main.tex]: I would clarify "clinical cases of malaria"

File: main.tex [hunk 11]
```diff
@@ -70,19 +70,102 @@
...
+%
+More recently an experimental hut trial in Tanzania found that PBO nets were more effective that traditional LLINS where pyrethroid resistance was prevalent \cite{protopopoff2018}, however, these bed-nets are more expensive than pyrethroid only LLINs \cite{churcher2016}.
```
Comment (EricRLucas) [inline main.tex]: that -> than

File: main.tex [hunk 12]
```diff
@@ -70,19 +70,102 @@
...
+%
+An important question, therefore, is can we predict where PBO net distribution could most impact vector populations and conversely, where there is no benefit over traditional LLINs \cite{protopopoff2018, toe2018}?
```
Comment (EricRLucas) [inline main.tex]: Again personal preference: "An important question is therefore whether we can predict where PBO net distribution could have most impact on vector populations ..."

File: main.tex [hunk 13]
```diff
@@ -70,19 +70,102 @@
...
 %%
 
+
 %%
-IR
-...FROM THE PROTOPOPOFF LANCET - "Cohort studies have shown that long-lasting insecticidal nets remain protective against malaria infection in areas of moderate insecticide resistance in Malawi8 and Kenya,9 whereas no reduction in incidence was observed after the distribution of these nets in Uganda.10"
+Here, we describe the population diversity, structure and insecticide resistance genetic marker status of the vector populations publicly available as the Ag1000G Project Phase 2 data release.
+%
+This includes whole genome sequencing data for 1142 mosquitoes, sequenced to an average depth of 30x coverage, from 16 natural populations in 13 sub-Saharan African countries, alongside 11 laboratory crosses.
```
Comment (EricRLucas) [inline main.tex]: I would say "comprises" rather than "includes" since you then provide an exhaustive list of what is included.

File: main.tex [hunk 14]
```diff
@@ -70,19 +70,102 @@
...
+%
+Analyses reveal the breadth of epidemiologically relevant demography and evolutionary histories that are now represented in these data.
+%
+Investigations into target site and metabolic resistance distributions across natural populations illustrate the power of this genomic resource to predictively inform malaria eradication efforts.
```
Comment (EricRLucas) [inline main.tex]: power -> potential ?

File: main.tex [hunk 15]
```diff
@@ -238,9 +311,7 @@ \subsection*{Population structure}
 %%
```
Comment (EricRLucas) [inline main.tex]: For some reason, I'm unable to comment on lines 280-310. On line 303, I'm not sure why we say "despite" here. Sure, they are both islands, but on different sides of Africa. Even if similar processes affected both of them, along with parallel phenotypic evolution, you would not expect them to look similar genetically.

File: main.tex [hunk 16]
```diff
@@ -299,14 +370,11 @@ \subsection*{Genetic diversity within populations}
 \end{figure}
```
Comment (EricRLucas) [inline main.tex]: Line 332: I would rephrase to explain (in a couple of words) more precisely what would be indicative of inbreeding (eg: high fractions of the genome with ROH, rather than just the fraction of the genome with ROH).

File: main.tex [hunk 17]
```diff
@@ -419,112 +510,93 @@ \subsection*{Insecticide resistance}
```
Comment (EricRLucas) [inline main.tex]: line 496: maybe -> may be

File: main.tex [hunk 18]
```diff
@@ -70,19 +70,102 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section*{Introduction}
 
+
 %%
-TODO
+During the 1950s, the widespread use of the insecticide DDT and the antimalarial drug chloroquine in the fight against malaria resulted in high levels of insecticide and drug resistance, demonstrating both the strength that anthropogenic pressure can exert and the speed at which selection can generate phenotypes to counteract it.
```
Comment (EricRLucas) [inline main.tex]: You asked for an alternative first sentence. How about this:
During the 1950s, the widespread use of the insecticide DDT and the antimalarial drug chloroquine in the fight against malaria resulted in high levels of insecticide and drug resistance [citations needed. For insecticide resistance, could cite review by Hemingway & Ranson: "Insecticide resistance in insect vectors of human disease." Annual review of entomology 45.1 (2000): 371-391.], highlighting the challenging nature or malaria eradication when both paraiste and vector are able to respond to rapidly evolve defenses against control interventions.

**Comment by cclarkson** [comment]
@EricRLucas  - many thanks, I'll wait for Nicks review then I'll incorporate your suggestions/corrections.

Regarding the Cas9 figure, it's on my list of things to fix so just leave it for now.

Cheers,
C

File: main.tex [hunk 19]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
+Genomic variation and meta-data are now available to download via FTP from www.malariagen.net for 16 distinct populations of mosquitoes, sampled from 13 countries across Sub-Saharan Africa. 
 %
-We used the Phase 2 dataset to extend the study of diversity in the \textit{dsx} Cas9 target region to more samples and new populations. 
+We defined 61\% (140Mbp) of the genome to be accessible (where we confidently call SNPs) and from these accessible regions we identified 57,837,885 high-quality SNPs, an average of one variant allele every 1.9 bases of accessible genome.
 %
-No further variant sites were discovered, and the mutant allele at the known site was not found in any of the additional populations in phase 2.
+Ag1000G Phase 2 also sees the release of 11 colony crosses, parents and offspring, we used the Mendelian errors detectable from parent offspring trios in the quality control process for wild caught samples but these also provide a valuable resource in calibration and validation of population genetic analyses more generally.
```
Comment (EricRLucas) [inline main.tex]: I would rephrase and split this sentence: "Ag1000G Phase 2 also sees the release of data the parents and offspring of 11 colony crosses. We used ...."

File: main.tex [hunk 20]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
 %
-Since we found no other variant sites, there is no evidence from the 1142 samples in Phase 2 of naturally-occurring variation that could disrupt this gene drive target. 
-
-The occurrence of alternative splicing at the site of this gene drive target may explain the low diversity in the region, and could impose purifying selection pressures on a wider area than these 23bp.
+"raw" data containing all genotype calls, "pass" data containing calls that pass our hard filtering regime from accessible regions of the genome and "biallelic" data, potentially the most useful as both the smallest and the standard input for most population genetic analyses.
 %
-We investigated variation in the wider region around the target and found that it was lower than the mean variation in other genes on chromsome 2R (\ref{dsx1}). 
+Biallelic data are the subset of pass data where variants have two alleles, a reference allele and a single alternative allele, in Phase 2 this subset consists of 44,076,901 SNPs.
```
Comment (EricRLucas) [inline main.tex]: I would split this in two, starting a new sentence just before "In Phase 2 ..."

File: main.tex [hunk 21]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
 %
-A study using  meta-analysis to parameterise mathematical models suggests that PBO bednets could avert up to 0.5 clinical cases per person per year in some locations \cite{churcher2016}.
+These analyses found much lower isolation by distance in \textit{An. gambiae} than \textit{An. coluzzii}, suggesting long distance gene flow, and therefore long distance migration in the former but not the latter species (concordant with conclusions made from the results of a study using time-series analyses \cite{dao2014}).
 %
-More recently an experimental hut trial in Tanzania found that PBO nets were more effective that traditional LLINS where pyrethroid resistance was prevalent \cite{protopopoff2018}, however, these bed-nets are more expensive than pyrethroid only LLINs \cite{churcher2016}.
+\textit{An. gambiae’s} genetic connectivity and \textit{coluzzii's} lack thereof, have important connotations for CRISPR Cas9 gene-drive mediated vector control as high migration in \textit{gambiae} may make these populations more susceptible to introgression of gene-drive resistance loci.
```
Comment (EricRLucas) [inline main.tex]: The 's shouldn't be italicised. But I would anyway probably rephrase this as "The higher genetic connectivity found in An. gambiae compared to An. coluzzii has important implications for CRISPR ... more susceptible to the spread of gene-drive resistance loci"

File: main.tex [hunk 22]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
+%
+Gene-drive resistance can be caused by genetic diversity in the Cas9 binding target, natural variation here could inhibit the drive mechanism and thus constitute a resistant genotype that could spread and resist control.
 %
-An important question, therefore, is where deployment of PBO nets could most impact vector populations and conversely, where there is no benefit over traditional LLINs \cite{protopopoff2018, toe2018}.
+A proposed gene-drive target for release into natural populations is \textit{dsx}, fortunately scans of variation found no evidence in Phase 2 samples of any naturally-occurring variation that could disrupt Cas9 binding \cite{kyrou2018}.
```
Comment (EricRLucas) [inline main.tex]: I would rephrase. Perhaps: "A gene-drive construct targeting the dsx gene has already been proposed for release into natural populations (Kyrou et al 2018). Using the Phase 2 resources, we found no evidence of novel naturally-occuring variation that could disrupt Cas9 binding.

File: main.tex [hunk 23]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
+A proposed gene-drive target for release into natural populations is \textit{dsx}, fortunately scans of variation found no evidence in Phase 2 samples of any naturally-occurring variation that could disrupt Cas9 binding \cite{kyrou2018}. 
 %%
 
+
 %%
+We also utilised the deep-sequencing and broad geographic coverage of these data to demonstrate a genomic approach to insecticide resistance management.
+%
+With the reliance on pyrethroid insecticides for ITNs and the ubiquity of pyrethroid resistance \cite{Hemingway2016}, it is essential for NMCPs to be able to access information about which regions might benefit from new (and more expensive \cite{churcher2016}) PBO bed-nets.
+%A putative 
 By combining genomic data on loci associated with pyrethroid target-site and metabolic resistance, we were able to create a snapshot of resistance topology across Africa.
```
Comment (EricRLucas) [inline main.tex]: I would add "putative" before "resistance topology"

File: main.tex [hunk 24]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
 %
 Results revealed that resistance topology fell broadly into three types (Figure \ref{ir}), each informative to the potential impact of PBO nets on vector populations:
+%to vector control
+\textbf{1.} Regions with only metabolic pyrethroid resistance loci, where the PBO nets may drastically reduce the resistance phenotype.
```
Comment (EricRLucas) [inline main.tex]: loci -> alleles?

File: main.tex [hunk 25]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
 %
-\textbf{2.} Regions with both high target-site and high metabolic resistance associated loci frequencies, many individuals in these regions carry markers for both modes of resistance.
+It is also likely that the four loci we analysed do not explain all phenotypic variation in all locations \cite{donnelly2016, mitchell2014}, and although all loci involve resistance associated genomic regions, the specific P450 CNVs used here have yet to undergo molecular validation \cite{lucas2019}.
+%to vector control
+In order to extract the maximum value from future genomic insecticide resistance monitoring we suggest a two stage approach.
 %
-The outcome of PBO nets is less intuitive in these regions, as the addition of the synergist can only reduce the metabolic resistance mediated fraction of the pyrethroid resistance phenotype.
+Firstly, sampling should be conducted on a wider geographic scale to improve understanding of population structure and diversity across sub-Saharan Africa, ensuring that genetically representative sentinel collection sites can be chosen and established.
```
Comment (EricRLucas) [inline main.tex]: Could potentially citethe GAARDian paper here: Sedda et al. Improved spatial ecological sampling using open data and standardization: an example from malaria mosquito surveillance16J. R. Soc. Interface. It describes the design of a sampling framework that tries to determine the geographical scale on which future collections should be performed. Although this paper necessarily doesn't preach the collection of data on a wide scale, rather the attempt to work out the best scale for local collections. so maybe not appropriate. See what you think. Happy to discuss if you like.

File: main.tex [hunk 26]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
 %%
 
 %%
-It must be reiterated that our results are a snapshot of predicated pyrethroid resistance at the time of sample collection (Table \ref{table:sampling_locations}).
-%
-It is likely that the four loci we analysed do not explain all phenotypic variation in all locations \cite{donnelly2009}, and although all loci involve resistance associated genomic regions, the specific P450 CNVs used here have yet to undergo molecular validation (lucas et al 2019).
+MalariaGen is committed to sequencing and releasing data that will make this approach to genomic insecticide resistance surveillance possible. 
 %
-To be most informative, future analysis should be routinely run on recently collected and sequenced samples and resistance associated loci regularly updated;
+Phase 3 of Ag1000G will see genome sequencing and related meta data released for 28 natural populations of Anopheles gambiae, coluzzii (@@?and arabiensis), from @@n countries, filling gaps in current coverage including populations from central African.
```
Comment (EricRLucas) [inline main.tex]: Why the question for arabiensis? Is it not certain that it will be included in phase 3?

African -> Africa

File: main.tex [hunk 27]
```diff
@@ -419,112 +510,110 @@ \subsection*{Insecticide resistance}
...
+These data will be curated and released in the same format as Ag1000G, generating unparalleled resources for vector management tools including genomic surveillance. 
 %%
 
 
+%%
+Here, in this brief overview of Ag1000g Phase 2, we provide examples of how this rich resource of deep genome sequencing data can quickly, easily and relatively cheaply edify gene-drive vector control development, PBO net distribution and insecticide choice.
+%
+Our findings emphasize the value of large publicly available genomic data that can inform many aspects of malaria control while filling gaps in our understanding of mosquito evolution and elucidating basic biological/ecological processes.
+%
+Expanding and improving these repositories with higher sampling coverage and faster data turnaround can only strengthen the vector control armamentarium in the fight against malaria.
```
Comment (EricRLucas) [inline main.tex]: "armamentarium" You know you're not allowed to make up words, right Chris? :-)

File: main.tex (hunk 2 above)
Comment (cclarkson) [reply main.tex]: changed

File: main.tex (hunk 4 above)
Comment (cclarkson) [reply main.tex]: removed citation for now

File: main.tex (hunk 5 above)
Comment (cclarkson) [reply main.tex]: cool, fixed.

File: main.tex (hunk 6 above)
Comment (cclarkson) [reply main.tex]: changed

File: main.tex (hunk 7 above)
Comment (cclarkson) [reply main.tex]: fixed

File: main.tex (hunk 8 above)
Comment (cclarkson) [reply main.tex]: changed

File: main.tex (hunk 9 above)
Comment (cclarkson) [reply main.tex]: sorted

File: main.tex (hunk 10 above)
Comment (cclarkson) [reply main.tex]: sorted

File: main.tex (hunk 12 above)
Comment (cclarkson) [reply main.tex]: sorted

File: main.tex (hunk 13 above)
Comment (cclarkson) [reply main.tex]: It includes other things - selection scans, haplotypes etc

File: main.tex (hunk 15 above)
Comment (cclarkson) [reply main.tex]: No, no genetically, but the effects on diversity, demography etc. are often shared between island populations of organisms.

File: main.tex (hunk 16 above)
Comment (cclarkson) [reply main.tex]: fixed

File: main.tex (hunk 17 above)
Comment (cclarkson) [reply main.tex]: fixed

File: Abstract_Intro_based_on_Pf6 [hunk 28]
```diff
@@ -0,0 +1,60 @@
...
+MalariaGen is a data sharing network that enables groups around the world to the use genomic tools to study malaria disease and transmission.
+
+Here we describe a new release of curated whole-genome variation data on 1,142 malaria vector mosquitoes, 
+sampling from 16 populations in 13 sub-Saharan African countries as part of Phase 2 of the Anopheles gambiae 1000 Genomes Project.
+
+High-quality genotype calls on ~58 million single nucleotide polymorphisms (SNPs) were produced using a standardised analysis pipeline.
+
+The expanded sampling of the Phase 2 data set allowed investigations into population structure and genetic diversity which highlight medically relevant demographic features differentiated between geographic regions.
+
+For example the island populations, novel to Phase 2 reveal divergent evolutionary histories, high diversity and low Tajima's D in Bioko island samples but the inverse in those from the island of
```
[the remark this answers is not in the document]
Comment (cclarkson) [reply Abstract_Intro_based_on_Pf6]: In this document I was just playing around with the style of abstract that was used in the Pf6 paper. I was holding off writing the abstract proper until the rest of the paper has been locked down.

File: Abstract_Intro_based_on_Pf6~ [hunk 29]
```diff
@@ -0,0 +1,58 @@
+Phase 2
```
[the remark this answers is not in the document]
Comment (cclarkson) [reply Abstract_Intro_based_on_Pf6~]: fixed

File: main.tex [hunk 30]
```diff
@@ -70,19 +70,102 @@
...
+
+
+%%
+Currently, surveillance of insecticide resistance in natural malaria vector populations involves two approaches:
+%
+resistance susceptibility testing with insecticide coated paper (\textit{e.g.} WHO tube assay \cite{world2016test}) and the detection of genetic loci associated with resistance via molecular assays (\textit{e.g.} PCR \cite{Ranson2000} and TaqMan \cite{bass2007}).
+%
+Measuring the insecticide susceptibility phenotypes of live mosquitoes samples provides an almost real-time picture of the strength of resistance (perhaps just one generation removed) and insecticides can be tested with no prior knowledge about potential modes of resistance or of underlying causative genetic resistance loci.
+%
+Monitoring natural populations is important but this approach is laborious, plus it requires expensive environmentally controllable spaces (humidity/temperature/light) to enable mosquito breeding and to ensure repeatability of testing \cite{kleinschmidt2018}.
```
[the remark this answers is not in the document]
Comment (cclarkson) [reply main.tex]: formalised

File: main.tex [hunk 31]
```diff
@@ -70,19 +70,102 @@
...
+A deeper understanding of resistance evolution and the ability to parameterise models for resistance spread provided by genomic surveillance of natural populations facilitates a move into predictive rather than reactive insecticide resistance management.
 %%
 
+
 %%
-IR
-...FROM THE PROTOPOPOFF LANCET - "Cohort studies have shown that long-lasting insecticidal nets remain protective against malaria infection in areas of moderate insecticide resistance in Malawi8 and Kenya,9 whereas no reduction in incidence was observed after the distribution of these nets in Uganda.10"
+The pervasive increase in pyrethroid resistance detected across Sub-Saharan Africa is raising fears of LLIN failures leading to increases in malaria \cite{churcher2016, Hemingway2016}.
+%
+To ameliorate the risk of vector control failure, pyrethoid LLINs containing the synergist piperonyl butoxide (PBO) have been in development and testing for some time \cite{darriet2011, protopopoff2018}.
```
[the remark this answers is not in the document]
Comment (cclarkson) [reply main.tex]: As far as I know "Ameliorate risk" is a fairly standard usage.

File: main.tex [hunk 32]
```diff
@@ -70,19 +70,102 @@
...
+%
+The output of Ag1000G includes whole genome variation, phased haplotypes, meta-data and genomic scans for signatures of selection, all of which is freely available to analyse under MalariaGen's guiding principles on data sharing \cite{network2008, chokshi2006, parker2009}.
+%
+Phase 1 of the project saw sequencing data released from 765 wild caught \textit{Anopheles} \cite{Ag1000gConsortium2017}, which have already been used to address a wide range of scientific questions regarding malaria control that would be difficult or impossible to investigate without access to this kind of data repository.
+%
+Ag1000G Phase 1 data have been used for highlighting new potential signals of insecticide resistance (Harding biorxiv?), to develop new methods for selection scans using deep-learning \cite{xue2019}, to identify genomic targets for CRISPR-Cas9 mediated gene drive vector control \cite{kyrou2018} and to improve demographic modelling of vectors \cite{khatri2018}.
```
[the remark this answers is not in the document]
Comment (cclarkson) [reply main.tex]: The paper is actually called "... with deep learning"

File: main.tex [hunk 33]
```diff
@@ -70,19 +70,102 @@
...
+A study using  meta-analysis to parameterise predictive mathematical models suggests that PBO bednets could avert up to 0.5 clinical cases per person per year in some locations \cite{churcher2016}.
+%
+More recently an experimental hut trial in Tanzania found that PBO nets were more effective that traditional LLINS where pyrethroid resistance was prevalent \cite{protopopoff2018}, however, these bed-nets are more expensive than pyrethroid only LLINs \cite{churcher2016}.
+%
+An important question, therefore, is can we predict where PBO net distribution could most impact vector populations and conversely, where there is no benefit over traditional LLINs \cite{protopopoff2018, toe2018}?
 %%
+
+
+%%
+Here, we describe the population diversity, structure and insecticide resistance genetic marker status of the vector populations publicly available as the Ag1000G Project Phase 2 data release.
```
[the remark this answers is not in the document]
Comment (cclarkson) [reply main.tex]: changed

File: main.tex [hunk 34]
```diff
@@ -330,47 +398,70 @@ \subsection*{Isolation by distance}
...
 %
-An observation of strong structure near this gene thus squares with theory predicting that population structure can be increased at sites linked to recent selective sweeps \cite{slatkin1998genetic}.
+We used the Phase 2 dataset to extend the study of diversity in the \textit{dsx} Cas9 target region to more samples and new populations. 
 %
-However, we observed the opposite pattern in \textit{An. coluzzii}, with neighbourhood size estimates near \textit{Vgsc} instead strongly inflated relative to surrounding regions.
+No further variant sites were discovered, and the mutant allele at the known site was not found in any of the additional populations in phase 2.
 %
-This reflects high levels of genetic differentiation among some nearby populations of \textit{An. coluzzii}, which results in a flat or even negative slope of $F_{st}$ as a function of distance (Supplementary Figure \ref{fig:vgsc_ibd}).
+The mutant allele was present at 2.9\% frequency in Phase 1, and is now at 2.5\% frequency in Phase 2, being found in \textit{An.coluzzii} from Angola (26.5\% frequency), and \textit{An. gambiae} from Cameroon (1.3\%) and Gabon (5.8\%).
```
[the remark this answers is not in the document]
Comment (cclarkson) [reply main.tex]: yep, fixed

File: main.tex [hunk 35]
```diff
@@ -330,47 +398,70 @@ \subsection*{Isolation by distance}
...
 %
 There are no published markers for P450-based pyrethroid resistance, however it is known that P450 resistance can be achieved via over-expression of P450 genes, which in turn can be driven by copy number amplification of the relevant gene within the genome \cite{riveron2014, muller2008}.
 %
-We scanned the genomes of the mosquitoes in the Phase 2 cohort for evidence of copy number variation, described in more detail in (Lucas et al. 2019), these CNV results are released as part of the Phase 2 data resource.
+We scanned the genomes of the mosquitoes in the Phase 2 cohort for evidence of copy number variation, described in more detail in \cite{lucas2019}, these CNV results are released as part of the Phase 2 data resource.
 %
 At two P450 loci in particular, the \textit{Cyp6p/aa} gene cluster and the \textit{cyp9k1} gene (referred to hereafter as \textit{Cyp6} and \textit{Cyp9} respectively), we found a high frequency of copy number variation in multiple populations.
```
[the remark this answers is not in the document]
Comment (cclarkson) [reply main.tex]: Hi @hardingnj, I might need some clarification here - do you mean we should describe other ways of acquiring P450 pyrethroid resistance?

As an aside, couldn't the CMS signal be driven by CNVs?

File: main.tex (hunk 19 above)
Comment (cclarkson) [reply main.tex]: cool, fixed

File: main.tex (hunk 20 above)
Comment (cclarkson) [reply main.tex]: fixed

File: main.tex (hunk 21 above)
Comment (cclarkson) [reply main.tex]: fixed

File: main.tex (hunk 23 above)
Comment (cclarkson) [reply main.tex]: sorted

File: main.tex (hunk 22 above)
Comment (cclarkson) [reply main.tex]: Much better, thanks.

File: main.tex (hunk 24 above)
Comment (cclarkson) [reply main.tex]: changed

File: main.tex (hunk 25 above)
Comment (cclarkson) [reply main.tex]: This paper is great, however the approaches suggested have not been employed yet by MalariaGen and I was trying to tie that paragraph with the following one.

Perhaps it could be referenced elsewhere, let's discuss later this week.

File: main.tex (hunk 26 above)
Comment (cclarkson) [reply main.tex]: I was unclear exactly what populations will be included in the P3 release, but this can be sorted later.

Fixed "African"

File: main.tex (hunk 18 above)
Comment (cclarkson) [reply main.tex]: Thanks, I've made some changes based on your suggestion.
